applications/TopologyOptimizationApplication/tests/Small_Cantilever/Optimization_Parameters.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
Can be appended to the project parameters?

Reply:
We wanted to set the optimization parameters in a different json file as it is done in the shape optimization

Reply:
Okay (in that case I would like an explanation from the team of shape optimization, because is inconsistent with the others apps)

Reply:
Is it possible to leave it like this for now?

Reply:
OK, but my other comments are still not resolved

Reply:
Simple reason for having the optimization settings separate from the actual project parameters is, that we might use several project parameter files to define the individual analyses of the response functions (imagine e.g optimizing for several load cases). 

Separating the settings of optimization and analyses makes the setup easier since a standard project parameters files for the analyses can simply be reused, and it also allows to easily run the involved analysis standalone, without optimization.

TLDR: its more flexible and easier to set up.

Reply:
Now it is resolved I am happy